### ISSUE TITLE
Right click on bar throws error

### ISSUE THREAD

**Issue by webberig** [Sonaza/Experiencer#8]
A freshly installed addon throws following error when I right-click the bar:

```
Message: Interface\AddOns\Experiencer\modules\reputation.lua:488: bad argument #1 to 'tinsert' (table expected, got nil)
Time: Mon Jul 31 08:50:06 2017
Count: 1
Stack: Interface\AddOns\Experiencer\modules\reputation.lua:488: bad argument #1 to 'tinsert' (table expected, got nil)Interface\SharedXML\SharedBasicControls.lua:204: in function `origErrorHandler'
...rface\AddOns\TradeSkillMaster\Debug\ErrorHandler.lua:317: in function <...rface\AddOns\TradeSkillMaster\Debug\ErrorHandler.lua:300>
[C]: ?
[C]: in function `tinsert'
Interface\AddOns\Experiencer\modules\reputation.lua:488: in function `GetReputationsMenu'
Interface\AddOns\Experiencer\modules\reputation.lua:372: in function `GetOptionsMenu'
Interface\AddOns\Experiencer\core.lua:1243: in function `OpenContextMenu'
Interface\AddOns\Experiencer\core.lua:1309: in function <Interface\AddOns\Experiencer\core.lua:1285>

Locals: errorMessage = "Interface\AddOns\Experiencer\modules\reputation.lua:488: bad argument #1 to 'tinsert' (table expected, got nil)"
DisplayMessageInternal = <function> defined @Interface\SharedXML\SharedBasicControls.lua:187
MESSAGE_TYPE_ERROR = 0
```

**Comment by Sonaza** [comment]
Seems there is an error with reputation menu generation logic.

What reputations does your character have, is it a brand new character? It's a little weird that I haven't encountered this myself yet.

**Comment by webberig** [comment]
I created a new character which is now level 2. After that, I installed experiencer and got the error.
As you suspected, the error doesn't occur with "older" characters. 

Thanks for the tip, I can now configure it while logged in with another character! :-)

**Comment by Sonaza** [comment]
Trying to replicate the error, what was the character in question, like the race and the class?

**Comment by webberig** [comment]
It was a Tauren Druid. The problem still occurs now, I'm level 11...

2017-07-31 21:00 GMT+02:00 Sonaza <notifications@github.com>:

**Comment by Sonaza** [comment]
Seems the error is related to weird reputation category listing on the tauren character. API says a reputation category header is a subheader when the primary header is still missing. What I mean, when you discover more factions the "Classic" category header will appear and will separate reputations from later expansions. What is stranger this error doesn't happen on at least a few other Alliance races that I tested too.

It is fixable when I have the time to look into it but as a temporary fix you could try and see if discovering new "non-classic" reputations will help for the time being.

**Comment by webberig** [comment]
The problem went away around level 12 I think, so I don't see this as a
major bug...

2017-08-02 8:48 GMT+02:00 Sonaza <notifications@github.com>:

**Comment by Sonaza** [comment]
Fixed this in update 2.3.1 now.
